core/coretypes/include/coretypes/baseobject.h

```diff
@@ -39,11 +37,11 @@ static constexpr IntfID UnknownGuid = { 0x00000000, 0x0000, 0x0000, { { 0xC0, 0x
 struct IUnknown
 {
     /// @privatesection
-    DEFINE_EXTERNAL_INTFID(UnknownGuid)
```

Comment:
`IUnknown` must stay compatible with Microsoft COM where its IntefaceID / Guid is defined as
```
 { 0x00000000, 0x0000, 0x0000, { { 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46 } } };
```